src/DynamoCoreWpf/Views/Core/AnnotationView.xaml

```diff
@@ -284,66 +285,6 @@
         </Style>
     </UserControl.Resources>
 
-    <UserControl.Triggers>
```

Comment:
Can you comment in this PR the reason to removal just for future references

Reply:
add a small comment about this in the description